cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -176,9 +177,8 @@ <h3 class="o-post-preview_title">
                       </a>
                 {% endif %}
             {% endfor %}
-            </div>
```

Comment:
Why was this closing div removed? Looking at the source file it doesn't look properly close anymore?

Reply:
Hrm. Looks like an accident. I don't recall seeing that in my `diff`s so I don't know how that happened.